src/Apps/Order/redirects.tsx

```diff
@@ -186,3 +205,39 @@ export const redirects: RedirectRecord<{
     },
   ],
 }
+
+graphql`
+  fragment redirects_order on Order {
+    id
+    mode
+    state
+    ... on OfferOrder {
+      myLastOffer {
+        id
+        createdAt
+      }
+      lastOffer {
+        id
+        createdAt
+      }
+    }
+    requestedFulfillment {
+      __typename
+    }
+    lineItems {
+      edges {
```

Comment:
It looks like the code only ever uses the first edge, should this reflect that?

Reply:
I can't figure out how to do that with this connection. Would it require a MP change?

Reply:
Hmm, indeed. A connection typically accepts a `first: N` parameter, but I don’t see it in the schema. Perhaps Exchange doesn’t even support it, because the thinking might be that an order will never have too many line items? 🤔 /cc @ashkan18

Reply:
hmmm... Exchange has it in it's schema here https://github.com/artsy/exchange/blob/master/_schema.graphql#L73, https://github.com/artsy/exchange/blob/master/_schema.graphql#L87 but this maybe missing in MP?
Looking at MP, it seems this connection ends up having a [`PageInfo`](https://github.com/artsy/metaphysics/blob/master/_schema.graphql#L6136) which [seems to](https://github.com/artsy/metaphysics/blob/master/_schema.graphql#L6225) implements things differently

Reply:
It indeed seems like MP’s versions of the `lineItems` fields does not take any parameters 😐

The `pageInfo` field is unrelated to this, though, that’s nested inside the connection type.